frontend/tests/e2e/features/common/index.ts

```diff
@@ -188,17 +188,19 @@ When('starts fishbowl', () => {
       response: []
     }
   });
+});
 
-  cy.contains('Start the fishbowl').click();
-
+When('starts fishbowl', () => {
```

Comment:
Shouldn't you also do a click on the start fishbowl button?

Reply:
It is doing so in line 201